src/dotnet/commands/dotnet-restore/LocalizableStrings.resx

```diff
@@ -172,4 +172,19 @@ This is equivalent to deleting project.assets.json.</value>
   <data name="CmdInteractiveRestoreOptionDescription" xml:space="preserve">
     <value>Allows the command to stop and wait for user input or action (for example to complete authentication).</value>
   </data>
+  <data name="CmdLockedModeOptionDescription" xml:space="preserve">
+    <value>Do not allow updating project lock file.</value>
+  </data>
+  <data name="CmdLockFilePathOption" xml:space="preserve">
+    <value>ProjectLockFile_Path</value>
+  </data>
+  <data name="CmdLockFilePathOptionDescription" xml:space="preserve">
+    <value>Project lock file path to be used to write lock file.</value>
+  </data>
+  <data name="CmdReevaluateOptionDescription" xml:space="preserve">
+    <value>Force restore to reevaluate all dependencies even if a lock file already exists.</value>
```

Comment:
This should say Forces... and reevaluate should be re-evaluate, I believe.

Reply:
"Forces restore to re-evaluate all dependencies even if a lock file already exists."

(my dictionary has the dash, if we have prior art without it and @mairaw says it's OK, then I'm good)

Reply:
I'd say this is without the -. This is what our style guide says:
In general, don’t include a hyphen after the following prefixes unless omitting the hyphen could confuse the reader. (and re- is one of them)

Reply:
Ok, I'll go with `Forces restore to reevaluate all dependencies even if a lock file already exists.`